app/src/main/java/com/zeapo/pwdstore/pwgenxkpwd/CapType.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+package com.zeapo.pwdstore.pwgenxkpwd
+
+enum class CapType {
```

Comment:
Please use a `sealed class` here

Reply:
What would be the advantage of using a sealed class here? An enum is simpler and offers some basic functionality like `ordinal()` an `valueOf()`. Converting to a sealed class will only make it more complex.

Reply:
sealed classes will make things easier should any kind of hierarchy ever come to play here. If you're confident that will never be the case, feel free to keep the enums.